src/components/page-configuration/style.css

```diff
@@ -1,26 +1,13 @@
-.scrim .delete-app {
-  background-color: var(--eds_ui_background__default);
-  display: grid;
-  grid-gap: var(--eds_spacing_medium);
+.delete-app-content {
+  width: 25rem;
+  min-width: 15rem;
+  margin: auto;
   padding: var(--eds_spacing_medium);
-  border-radius: var(--eds_spacing_x_small);
-  width: 90vw;
+  padding-top: 0;
+  overflow-y: auto;
 }
-
-.delete-app .header-actions {
-  display: grid;
-  grid-template-columns: 1fr auto;
-  align-items: center;
-}
-
-@media (min-width: 20rem) {
-  .scrim .delete-app {
-    width: 18rem;
-  }
-}
-
-@media (min-width: 50rem) {
-  .scrim .delete-app {
-    width: 25rem;
+@media (max-width: 30rem) {
```

Comment:
The smallest screen size should have the initial styling. Media queries should be used to accommodate larger screens using `min-width`